src/app/mission/[id]/detail/page.tsx

```diff
@@ -1,19 +1,34 @@
 'use client';
 
+import { useRouter } from 'next/navigation';
 import MissionHistoryTab from '@/app/mission/[id]/detail/MissionHistoryTab';
+import Dialog from '@/components/Dialog/Dialog';
 import Header from '@/components/Header/Header';
 import Tab from '@/components/Tab/Tab';
+import useModal from '@/hooks/useModal';
 import { css } from '@styled-system/css';
 
 export default function MissionDetailPage() {
+  const { isOpen, openModal, closeModal } = useModal();
```

Comment:
마이너 ) openModal 이라는 변수명이 추론하기 어려워서 openDeleteDialog 로 변경해주는건 어떤지 궁금해요~
```suggestion
  const { isOpen, openModal : openDeleteDialog , closeModal : closeDeleteDialog } = useModal();
```

Reply:
c34ea91472ea0dee1d4d502b609f6190431fa006 에서 수정했습니다!